source/delegate-factory/package.json

```diff
@@ -14,12 +14,19 @@
     "ganache": "ganache-cli -p 8545 --gasLimit 0xfffffffffff --time '2017-05-10T00:00:00+00:00'",
     "hint": "yarn solhint \"./contracts/**/*.sol\"",
     "lint": "yarn eslint \"./test/**/*.js\"",
-    "test": "truffle test"
+    "cp_migration_flat": "mkdir flatten/; cp contracts/Migrations.sol flatten/",
+    "test": "truffle test",
+    "flatten": "truffle run flatten",
+    "migrate": "yarn cp_migration_flat; truffle migrate --skip-dry-run",
```

Comment:
should this be 

yarn cp_migration_flat &&  truffle migrate --skip-dry-run or is the first one failing not a blocker for the second?

Reply:
not a blocker, but probably worth putting in.